mobile/src/features/ReviewTx/useCases/ReviewTxScreen/ReviewTx/Overview/OverviewTab.tsx

```diff
@@ -730,6 +730,7 @@ const useShowOperationsNotice = (operations: Operations) => {
   const {openModal} = useModal()
   const strings = useStrings()
   const screenHeight = useWindowDimensions().height
+  const hasShownModal = React.useRef(false)
```

Comment:
i think this is redundant with useQuery. Maybe the api of useQuery changed in the new version. Can you have a look?